project-rome-docs/objectivec-api/remotesystems.commanding/MCDRemoteSystemAppRegistrationPublishResult.md

```diff
@@ -0,0 +1,20 @@
+---
+title: MCDRemoteSystemAppRegistrationPublishResult
+description: A class communicates the async result of publishing remote system app information for an account.
+keywords: microsoft, windows, iOS, iPhone, objectiveC, connected devices, Project Rome
+---
+
+# class `MCDRemoteSystemAppRegistrationPublishResult` 
+
+```
+@interface MCDRemoteSystemAppRegistrationPublishResult : NSObject
+```  
+
+A class communicates the async result of publishing remote system app information for an account. The error statuses communicated through this result should be used by the app to retry publishing in the event of certain transient conditions like the network being unavailable.
```

Comment:
Should this paragraph start "This class communicates..." or "A class that communicates..." or something like that?